### PR TITLE
(Do not merge) Generate code with generic builder methods

### PR DESCRIPTION
**Pull request by antoyo**
https://github.com/gtk-rs/gir/pull/853

### CODE REVIEW

**Comment by EPashkin** [comment]
@antoyo Please add link to gir's PR in description

**Comment by EPashkin** [comment]
and run 'cargo fmt' to minimize changes

File: src/auto/application.rs [hunk 1]
```diff
@@ -178,9 +178,6 @@ pub trait GtkApplicationExt: 'static {
     #[cfg(any(feature = "v3_24", feature = "dox"))]
     fn get_property_screensaver_active(&self) -> bool;
 
-    #[cfg(any(feature = "v3_24_8", feature = "dox"))]
-    fn connect_query_end<F: Fn(&Self) + 'static>(&self, f: F) -> SignalHandlerId;
```
Comment (sdroege) [inline src/auto/application.rs]: Why is this removed?

Comment (antoyo) [reply src/auto/application.rs]: I don't know. That might be the result of another change than mine.

Comment (EPashkin) [reply src/auto/application.rs]: @antoyo  Seems you regenerated with old gir-files

**Comment by antoyo** [comment]
Closing as the gir PR was merged.
